test/e2e/scheduling/priorities.go

```diff
@@ -303,19 +303,40 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		// Apply 10 taints to first node
 		nodeName := nodeList.Items[0].Name
 
-		ginkgo.By("Trying to apply 10 (tolerable) taints on the first node.")
+		// First, create a set of tolerable taints (+tolerations) for the first node.
+		// We immediately defer the removal of these taints because addTaintToNode can
+		// panic and RemoveTaintsOffNode does not return an error if the taint does not exist.
```

Comment:
I would move the defer calls and this comment explaining why to right before the calls to `addTaintToNode`, there's nothing about generating them that's going to require a defer